etc/run-auth-ldap.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# see run-jupyterhub.sh for setting up local mail delivery
+export SIREPO_FROM_EMAIL='support@radiasoft.net'
+export SIREPO_FROM_NAME='RadiaSoft Support'
+export SIREPO_SMTP_PASSWORD='vagrant'
+# POSIT: same as sirepo.smtp.DEV_SMTP_SERVER
+export SIREPO_SMTP_SERVER='dev'
+export SIREPO_SMTP_USER='vagrant'
+export SIREPO_AUTH_METHODS='email:guest:ldap'
+export PYKERN_PKDEBUG_WANT_PID_TIME=1
+
+# primary cause of failed wheel generation
+pip install --upgrade pip
+echo "upgraded pip"
+
+# install prerequisites
+sudo yum groupinstall "Development tools"
+sudo yum install openldap-devel python-devel
+echo "installed prerequisites"
+
+# secondary cause of failed wheel generation
+pip install --upgrade setuptools wheel
+echo "installed wheel setup"
+
+# main installs
+pip install python-ldap
+sudo yum -y install openldap-clients openldap-servers
+echo "installed ldap"
+
+sudo systemctl start slapd
```

Comment:
I'm not sure we want to enable slapd, start should be good enough. We can talk about making this script idempotent later.